infra/proposer_test.go

```diff
@@ -0,0 +1,37 @@
+package infra_test
+
+import (
+	"github.com/guoger/stupid/infra"
+	"github.com/hyperledger/fabric-sdk-go/pkg/fab/mocks"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Proposer", func() {
+
+	const testAddress = "127.0.0.1:0"
+	const invalid_addr = "127.0.0.1:0"
+
+	var addr string
+	crypto := &infra.Crypto{}
+
+	BeforeEach(func() {
+		srv := &mocks.MockEndorserServer{}
+		addr = srv.Start(testAddress)
+	})
+
+	Describe("Test CreateProposers", func() {
+		Context("CreateProposers", func() {
```

Comment:
since we *do* have a method `CreateProposers` (plural), which is **not** the one being tested here. We need this name to precisely match the target, therefore `CreateProposer` (singular)